src/routes/help/components.svelte

```diff
@@ -0,0 +1,238 @@
+	};
+
+	let title = 'svelte-calender';
+	let url = 'https://github.com/6eDesign/svelte-calendar';
+	let description = 'A lightweight date picker with neat animations and a unique UX';
+	let npm = 'svelte-calender';
+	let image = '';
+	let category = 'Forms & User Input';
+	let tags = ['components and libraries', 'time and date'];
+	let addedOn = '';
+
+	$: jsonSnippet = {
+		title,
+		url,
+		description,
+		npm,
+		tags,
+		image,
+		addedOn,
+		category
+	};
+
+	function ascending(a, b) {
+		return a.label > b.label ? 1 : -1;
+	}
+</script>
+
+<h1>How to submit a new component?</h1>
+<p>
+	To add a new component on the website, the process is rather simple. You have to add your
+	component in <a
+		href="https://github.com/svelte-society/sveltesociety.dev/blob/master/src/routes/components/components.json"
+	>
+		components.json
+	</a>
+</p>
+
+<h2>Forking the project</h2>
+<p>
+	You can fork the <a href="https://github.com/svelte-society/sveltesociety.dev/">
+		GitHub project
+	</a>, add your component and then propose a Pull request.
+</p>
+
+<h2>Edit the file</h2>
+<p>
+	You can edit and propose your changes <a
+		href="https://github.com/svelte-society/sveltesociety.dev/edit/master/src/pages/components/components.json"
+	>
+		directly in GitHub
+	</a>
+</p>
+
+<h2>What information should I give ?</h2>
+<p>
+	Each component is represented by a JSON Object. Use the generator below to generate the JSON
+	Object.
+</p>
+
+<h2>Generator</h2>
+<p><code>*</code> marked fields are required</p>
+<div class="json-generator">
+	<div class="input-wrapper">
+		<label for="title">Title*:</label>
+		<div>
+			<input id="title" type="text" required bind:value={title} />
+			<span class="input-helper">Name of the component</span>
+		</div>
+	</div>
+	<div class="input-wrapper">
+		<label for="url">URL:</label>
+		<div>
+			<input id="url" type="url" bind:value={url} />
+			<span class="input-helper">The URL where to find it</span>
+		</div>
+	</div>
+	<div class="input-wrapper">
+		<label for="desc">Description:</label>
+		<div>
+			<input id="desc" type="url" bind:value={description} />
+			<span class="input-helper">A short description of the component</span>
+		</div>
+	</div>
+	<div class="input-wrapper">
+		<label for="npm">NPM:</label>
+		<div>
+			<input id="npm" type="url" bind:value={npm} />
+			<span class="input-helper">The npm name of the component</span>
+		</div>
+	</div>
+	<div class="input-wrapper">
+		<label for="img">Image:</label>
+		<div>
+			<input id="img" type="url" bind:value={image} />
+			<span class="input-helper">An image to quickly view what your component is all about</span>
+		</div>
+	</div>
+	<div class="input-wrapper">
+		<label for="adden-on">Added On*:</label>
+		<div>
+			<input id="adden-on" type="date" required bind:value={addedOn} />
+			<span class="input-helper">
+				The date when the component have been added on the website (generally it’s today)
+			</span>
+		</div>
+	</div>
```

Comment:
I'm not sure that field should be editable

Reply:
I'm not sure about it either. Currently, the date is not being displayed anywhere. And I think there is either a plan to automate the process or remove this field altogether. Need some more thoughts on this.

Reply:
I don't think it matters either way. The user is going to copy/paste the result and can edit it then, so there's not much point to preventing them from editing it. As long as the default is the current date it's probably fine